src/common/tracer.h

```diff
@@ -0,0 +1,66 @@
+#ifndef TRACER_H_
+#define TRACER_H_
+
+#define SIGNED_RIGHT_SHIFT_IS 1
+#define ARITHMETIC_RIGHT_SHIFT 1
+
+#include <iostream>
+#include <arpa/inet.h>
```

Comment:
where is this include used?

Reply:
`iostream` will be removed, `arpa/inet.h` is for htons  dependencies

Reply:
i dont see htons in the code here

Reply:
`htons` are required by `jaeger` libraries I was facing issue compiling with jaeger so this solution I got from stack overflow.

Reply:
in this case please add a tracer.cc file and add all this stuff there.